src/ResponseFactory.php

```diff
@@ -115,6 +88,11 @@ public function lazy(callable $callback): LazyProp
         return new LazyProp($callback);
     }
 
+    public function always(callable $callback): AlwaysProp
```

Comment:
Given that "always" props are always evaluated, there isn't really any reason to use a closure. So I think we should accept anything here like a regular prop:

```php
'authName' => Inertia::always(Auth::user()->name),
```

That said, it would be nice to still allow using a closure, just because that's sometimes nice for code formatting, so I think we should still support that as well.